drivers/internal/USBDevice.h

```diff
@@ -141,21 +142,6 @@ class USBDevice: public  USBPhyEvents {
     */
     bool endpoint_add(usb_ep_t endpoint, uint32_t max_packet, usb_ep_type_t type, ep_cb_t callback = NULL);
```

Comment:
If you're changing to `Callback`, could you please avoid giving it `NULL` - change to `nullptr`. Will avoid issues with some optimisation work I'm doing on `Callback`. Will save me touching this again later.

(That does make it a 5.14 change - we're not putting any C++11-isms into 5.13 patches. But this doesn't look like a patch anyway).